File: objctemplate/objctemplate/classes/controllers/busines/TMPJMainViewController.m
//
//  TMPJMainViewController.m
//  objctemplate
//
//  Created by supertext on 15/3/11.
//  Copyright (c) 2015年 icegent. All rights reserved.
//

#import "TMPJMainViewController.h"
#import "TMPJWebViewController.h"
@interface TMPJMainViewController ()

@end

@implementation TMPJMainViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [self showAlertMessage:@"点击跳转" clickedAction:^(UIView *sender) {
        [self.navigationController pushViewController:[[TMPJWebViewController alloc] initWithLink:@"http://www.baidu.com" showToolBar:YES] animated:YES];
    }];
    [self setLeftReturnIcon];
    [self setRightBarTitle:@"test"];
}

-(void)leftItemAction:(id)sender
{
    [[TMPJLayoutViewController sharedController] showLeftViewController:YES];
}
-(void)rightItemAction:(id)sender
{
    [[TMPJLayoutViewController sharedController] showRightViewController:YES];
}
@end

File: swifttemplate/swifttemplate/classes/controllers/business/TMPJMainViewController.swift
//
//  TMPJMainViewController.swift
//  swifttemplate
//
//  Created by supertext on 15/3/11.
//  Copyright © 2016年 icegent. All rights reserved.
//

import EasyTools

class TMPJMainViewController: TMPJTableViewController {
    override func viewDidLoad() {
        super.viewDidLoad();
        self.view.backgroundColor=UIColor.whiteColor()
    }
    override func leftItemAction(sender: AnyObject!) {
        TMPJLayoutViewController.sharedController.showLeftViewController(true);
    }
}
